data/abilities/collection/4d9b079c-9ede-4116-8b14-72ad3a5533af.yml

```diff
@@ -6,7 +6,7 @@
   tactic: collection
```

Comment:
collection needs to be changed to execution

Reply:
I actually changed the tactic of this and another ability to discovery and applied what I thought was the appropriate technique